robottelo/config/base.py

```diff
@@ -1079,6 +1079,7 @@ def __init__(self):
         self.webdriver = None
         self.webdriver_binary = None
         self.webdriver_desired_capabilities = None
+        self.command_executor = None
```

Comment:
Please update robottelo.properties.sample with this new option

Reply:
Done